uwblueprint/sistering
# Fix admin posting card event date range

## Ticket link

Closes #616



## Implementation description
* Fixed admin posting card to only show single date if posting is an event
* ~~Fixed time range in volunteer posting card to show correct time (was offset by 4 hours the posting shift times)~~



## Steps to test
1. Go to `http://localhost:3000/admin`
2. Create event posting
3. Check posting card date is not shown as range anymore
4. Check volunteer posting card time range is correct



## What should reviewers focus on?
* Code correctness


## Checklist
